apps/src/sites/studio/pages/signup.js

```diff
@@ -115,18 +123,118 @@ window.SignupManager = function (options) {
 
     // Force teachers to explicitly accept terms of service.
     $("#user_terms_of_service_version").prop('checked', false);
+
+    logTeacherToggle(true);
+    lastUserType = "teacher";
+  }
+
+  function getEnvironment() {
```

Comment:
Another PR: It seems appropriate to move this into the FirehoseClient class, filling in the `environment` column if it is not provided (also `created_at`, etc.) [something I'll own doing].